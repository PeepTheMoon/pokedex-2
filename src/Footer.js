import React, { Component } from 'react'

export default class Footer extends Component {
    render() {
        return (
            <div className="footer">
                This iteration of Pokedex was made by Rachel Donahue @PeepTheMoon on GitHub
            </div>
        )
    }
}
