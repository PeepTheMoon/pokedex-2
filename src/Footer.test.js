import React from 'react';
import { shallow } from 'enzyme';
import Footer from './Footer.js';


it('renders single pokemon item details', () => {
    const wrapper = shallow(<Footer/>);
  
    expect(wrapper).toMatchSnapshot();
  });