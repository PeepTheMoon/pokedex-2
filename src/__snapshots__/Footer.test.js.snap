// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders single pokemon item details 1`] = `
<div
  className="footer"
>
  This iteration of Pokedex was made by Rachel Donahue @PeepTheMoon on GitHub
</div>
`;
